Umbraco-Uno/Creating-Content/How-To-Add-A-New-Font-Style/index.md

```diff
@@ -0,0 +1,36 @@
+---
+versionFrom: 8.0.0
+---
+
+# How to add a new font style through the backoffice
+
+If you find your self in need of a font that is not already included in Umbraco Uno, then you have the option to add one your self. In this article, you will learn how to do that.
+
+What you will have to do is to go to https://fonts.google.com/ select a font you like and click the box.
```

Comment:
Instead of adding the link directly, I think it would be great with an anchor instead!
What do you think? 🤔

Reply:
might be more pretty that way yeah